src/modules/launcher/Plugins/Microsoft.Plugin.Calculator/CalculateHelper.cs

```diff
@@ -50,5 +51,16 @@ public static bool InputValid(string input)
 
             return true;
         }
+
+        public static string ConvertLocale(string input, CultureInfo cultureInfo)
+        {
+            // German input has commas as number delimiters instead of points
+            if (cultureInfo.Name.StartsWith("de-"))
```

Comment:
This seems not the correct thing to do, we need to get the system settings, regardless of the cultureInfo.

Reply:
Do you mean `CultureInfo.CurrentCulture` instead of `CultureInfo.CurrentUICulture`?

Reply:
This is absolutely incorrect, comma is not used only German language but in a lot of other ones, especially in Europe. 
You should parse string as a number using the current culture and return that number, to get numbers, split `input` by spaces
`decimal.TryParse(input, NumberStyles.Number, CultureInfo.CurrentCulture, out decimal result)`

Reply:
@martinchrzan 
> decimal.TryParse(input, NumberStyles.Number, CultureInfo.CurrentCulture, out decimal result)

will take into account a user custom settings?

![image](https://user-images.githubusercontent.com/3206696/104461856-29debb80-55b0-11eb-95f2-efd602ef6326.png)

Reply:
Yes it will, I just retested that myself

Reply:
Yes, I think that is the difference between `CurrentCulture` (for numer formatting, etc.) and `CurrentUICulture` (language).